src/Microsoft.ComponentDetection.Orchestrator/Minimatch.cs

```diff
@@ -0,0 +1,843 @@
+﻿namespace Microsoft.ComponentDetection.Orchestrator;
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.RegularExpressions;
+
+/// <summary>
```

Comment:
Assuming your reference was latest, this would be from commit [cb4be48a55d64b3a40a745d4a8eb4d1b06507277](https://github.com/isaacs/minimatch/commit/cb4be48a55d64b3a40a745d4a8eb4d1b06507277)

Reply:
Let's also make sure to follow the terms of the [license agreement](https://github.com/isaacs/minimatch/blob/main/LICENSE) as well. This essentially means copying the license into this file.

Reply:
Will fix.

Reply:
Thanks for adding this in 😃